projects/pt1/python/torch_mlir_e2e_test/test_suite/reshape_like.py

```diff
@@ -379,6 +379,25 @@ def ViewDynamicExpandCollapseModule_basic(module, tu: TestUtils):
 
 # ==============================================================================
 
+class ViewDynamicExpandCollapseWithParallelUnknownDimModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([2, 3, -1, 5], torch.float32, True),
+    ])
+
+    def forward(self, a):
+        return a.view(2, -1, 6)
```

Comment:
Is this the right result size? If I understand correctly, the case you're handling here is `a.view(2, -1, 5)`, right?

Reply:
the input size is 2x3x`a`x5,  which `a` = 4, 120 elements totally. the output size is 2x`b`x6， when b is 10, the total elements is 120, too.

Reply:
Oh, that's really nice! thanks for the explanation